kolibri/core/assets/src/core-app/client.js

```diff
@@ -86,9 +86,19 @@ const client = options => {
       options.params[cacheBust] = cacheBust;
     }
   }
+  let mimeType = 'application/json';
+  // in case a File obj must be POSTed:
+  if (options.entity !== 'undefined' && options.method === 'POST') {
+    if (
+      Object.values(options.entity).filter(function(a) {
+        return typeof a === 'object';
+      }).length > 0
```

Comment:
```suggestion
      Object.values(options.entity).filter(a => typeof a === 'object').length
```

Reply:
ah, perfect, I thought we had the rule of not using ES6 syntax. I also prefer that one.